ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -30,6 +31,13 @@ public ReactTextView(Context context) {
     mDefaultGravityHorizontal =
       getGravity() & (Gravity.HORIZONTAL_GRAVITY_MASK | Gravity.RELATIVE_HORIZONTAL_GRAVITY_MASK);
     mDefaultGravityVertical = getGravity() & Gravity.VERTICAL_GRAVITY_MASK;
+
+    // Android's TextView crashes when it tries to relayout if LayoutParams are
+    // null; explicitly set the LayoutParams to prevent this crash. See:
+    // https://github.com/facebook/react-native/pulls/7011
+    setLayoutParams(new ViewGroup.LayoutParams(
+        ViewGroup.LayoutParams.WRAP_CONTENT,
```

Comment:
I'd feel much safer if we were using `LayoutParams(0, 0)` instead of `WRAP_CONTENT`

Also layout param object can be created once as a static variable and then reused instead of allocating new object every time we create new component